README.md

```diff
@@ -14,6 +14,17 @@
 [![Travis-CI Build Status (Swift-Linux)](https://img.shields.io/travis/antlr/antlr4.svg?label=Linux-Swift&branch=master)](https://travis-ci.com/github/antlr/antlr4)
 -->
 
+## Versioning
+
+ANTLR 4 supports 10 target languages, and ensuring consistency across these targets is a unique and highly valuable feature.
+To ensure proper support of this feature, each release of ANTLR is a complete release of the tool and the 10 runtimes, all with the same version.
+As such, ANTLR versioning does not strictly follow semver semantics:
+- a component may be released with the latest version number even though nothing has changed within that component since the previous release
+- major version is bumped only when new grammar features are released (such as ANTLR3 -> ANTLR4)
```

Comment:
It's not clear what grammar features mean here. For instance, in ANTLR 4.10 the new grammar feature `caseInsensitive` was added. Is it a new grammar feature?

Reply:
I'll tweak after merge. i'll call it "complete rewrites" or ...